sdk/communication/azure-communication-chat/pom.xml

```diff
@@ -93,6 +93,12 @@
       <version>5.9.3</version> <!-- {x-version-update;org.junit.jupiter:junit-jupiter-params;external_dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-http-netty</artifactId>
+      <version>1.14.2</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
+      <scope>test</scope>
+    </dependency>
```

Comment:
This shouldn't be required. `azure-communication-common` and `azure-communication-identity` should have use `current` tag as they both are in the same service group - `<!-- {x-version-update;com.azure:azure-communication-identity;current} -->`

Reply:
This workaround code to be removed when actual fix is in, tracking the removal task in item 3674953 in ADO

Reply:
changed scope to compile